src/points/PointsHome.test.tsx

```diff
@@ -124,17 +124,12 @@ describe(PointsHome, () => {
   })
 
   it('renders multiple sections', async () => {
-    const { getByTestId, queryByTestId, queryByText } = renderPointsHome()
+    const { queryByText } = renderPointsHome()
 
-    expect(getByTestId('PointsActivitySection-50')).toBeTruthy()
-    expect(getByTestId('PointsActivitySection-20')).toBeTruthy()
+    expect(queryByText('points.activityCards.createWallet.title')).toBeTruthy()
```

Comment:
```suggestion
    expect(getByText('points.activityCards.createWallet.title')).toBeTruthy()
```
you get better error messages with `getBy*` and truthy assertions if the test fails :D